src/legacy/core_plugins/vis_type_timelion/public/helpers/arg_value_suggestions.ts

```diff
@@ -84,7 +87,7 @@ export function getArgValueSuggestions() {
     es: {
       async index(partial: string) {
         const search = partial ? `${partial}*` : '*';
-        const resp = await savedObjectsClient.find({
+        const resp = await savedObjectsClient.find<IndexPatternAttributes>({
```

Comment:
I added an explicit dependency on the IndexPatternAttributes to track all places accessing the data plugin saved object. It should simplify further refactoring.

Reply:
@elastic/kibana-app-arch let me know it it's okay